### PR TITLE
Added Issue and PR template

### LINKED ISSUE #7: Setup issue and PR templates
Issue templates
- BUG REPORT
- FEATURE REQUEST

Pull Request templates should have
- issue being resolved
- what's done in this pr
- testing required, if, then
- screenshots if applicable
- logs

### PR DESCRIPTION
**Pull request by ShaileshKumar007**
This PR fixes #7

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    attributes:
+      label: Screenshot/ Video
+      description: If applicable, add screenshots to help explain your problem.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Desktop (please complete the following information)
+      description: |
+        Device - [e.g. iPhone6]
```
Comment (YashKumarVerma) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]: renew device to "system"
rename iPhone to a laptop / pc specific detail as the backend would run on a PC and not a mobile

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    env:
+      PORT: 3000
+      DATABASE_CONNECTION_STRING: 'mongodb://127.0.0.1:27017/community-classrooms'
+      DATABASE_PASSWORD: ''
+    services:
+      mongodb:
+        image: mongo
+        ports:
+          - 27017:27017
+    steps:
```
Comment (YashKumarVerma) [inline .github/workflows/tests.yml:16]: this file is not needed

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Desktop (please complete the following information)
+      description: |
+        System - [e.g. Laptop,PC]
+        OS - [e.g. Windows,Linux,MacOS]
+        Browser [e.g. chrome,safari]
+        Version [e.g. 10]
```
Comment (YashKumarVerma) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]: its the backend, we really do not worry about the laptop or browser.
Something like NodeJS Version, NPM version would be appreciated.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (ShaileshKumar007) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I've changed the description. Let me know if any further changes are required.

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
just remove the GitHub actions workflow file then we can merge this.

**Comment by ShaileshKumar007** [comment]
> just remove the GitHub actions workflow file then we can merge this.

I've made a PR related to this (Deleted GitHub actions workflow folder).

**Review by divyanshu-parihar — APPROVED** [review]

**Comment by ShaileshKumar007** [comment]
> just remove the GitHub actions workflow file then we can merge this.

Done. Plz review it and let me know if any further changes required

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**